agent_deploy/IBMCloud-Kubernetes-Service/install-agent-k8s.sh

```diff
@@ -77,11 +91,16 @@ function help {
     echo " -np : if provided, do not enable the Prometheus collector.  Defaults to enabling Prometheus collector"
```

Comment:
If possible try to order available options

Reply:
Talked with @alblasco about this and agreed in do so in a next PR